src/hound/index/index.go

```diff
@@ -57,7 +59,7 @@ type ExcludedFile struct {
 }
 
 func (s Snapshot) Open() (*Index, error) {
-	return Open(string(s))
+	return Open(string(s), "")
```

Comment:
Ah yes, thanks for spotting that.  I had originally ripped it out, but put it back in just in case I was missing something... and forgot to follow up on it.